HelpSource/Classes/DecodeB2.schelp

```diff
@@ -17,7 +17,7 @@ method::ar, kr
 
 argument::numChans
 
-Number of output speakers. Typically 4 to 8.
+Number of output speakers. Typically 4 to 8. This cannot be assigned to a SynthDef argument and must be a fixed nonzero positive integer value.
```

Comment:
I think the current language is clear enough, but that's also because IMO folks will only look at these docs once it blows up and not before (or not at all like me, which is why I opened the scsynth question 😬 ).

However, you can look at how the In* ugens indicate this, as well as PlayBuf which is a little different, to see if the message is clearer.

Reply:
Maybe something like: "Must be a strictly positive integer. Note: The number of channels in a synth graph is fixed at compile time. Since this cannot be modulated, it can't be assigned to a SynthDef argument" ?

Reply:
Thanks for your thoughts on this @sirlensalot and @cdbzb. It's a bit tricky because there is a distinction here between a non-modulatable argument (i.e., "ir" = "initialization-rate" = "scalar rate"), which can still be a synthdef arg, and an immutable argument (compiled into the ugen).

Note these differences:
```supercollider
(
SynthDef(\irArg, {arg level = 0.2;
	DC.ar(level).poll;
}).add;
)
// >> OK, DC is set to level default
Synth(\irArg, 0.1) // works, but output is still 0.2

(
SynthDef(\irArg, {arg level; // level has no default (nil)
	DC.ar(level).poll;
}).add;
) // >> ERROR: Non Boolean in test.


(
SynthDef(\chanArg, {arg nchan = 2;
	In.ar(0, nchan); // immutable UGen arg, can't be a SynthDef arg
}).add;
) // >> ERROR: Non Boolean in test.
```

The goal would be to clearly distinguish between these, while also being concise :)

> However, you can look at how the In* ugens indicate this, as well as PlayBuf which is a little different, to see if the message is clearer.

Good point, in reading these docs, they're also not very good wording :-/ So looks like this should be corrected in a number of Help files. All the more reason to get the terminology right and keep the phrasing concise.

Reply:
@mtmccrea @cdbzb since we agree on the functional change to `MultiOutUGen` I might suggest we restrict this PR to that, and I'll open a new one with just the docs changes? The motivation is to get the better error message out to developers sooner.

Reply:
Sure. But I do think the coverage of your changes is still useful, so how about we just update the arg descriptions in the docs with the wording @cdbzb suggested. So the full phrasing (with suggested formatting) would read:

"Must be a nonzero, positive integer. This is fixed when the SynthDef is compiled so can't be assigned to a SynthDef argument."

Expanding the scope to other UGens like `In` and `PlayBuf` and further refining the language can be a future PR.

Thanks for sticking with it!